go/enclave/db/interfaces.go

```diff
@@ -54,8 +56,10 @@ type HeadsAfterL1BlockStorage interface {
 	FetchLogs(blockHash common.L1RootHash) ([]*types.Log, error)
 	// FetchL2Head returns the current L2 chain head.
 	FetchL2Head() (*common.L2RootHash, error)
-	// StoreNewHeads saves the new chain heads alongside its rollup, receipts and logs.
-	StoreNewHeads(l1Head common.L1RootHash, rollup *core.Rollup, receipts []*types.Receipt, isNewRollup bool, isNewL1Block bool) error
```

Comment:
This method has been split into three:

* `StoreRollup`, to store the new rollup
* `UpdateL2HeadForL1Block`, to update the rollup associated with a given L1 block
* `UpdateL1Head`, to update the current L1 head block